### PR TITLE
IVP-41: Field Updates + Age to String Update

### PR DESCRIPTION
**Pull request by dandigangi**
New fields for about endpoint added : ID, age changed to string per API resstrictions, and manager

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -110,12 +110,17 @@ app.get('/about', (req, res) => {
...
     name: 'Dan',
-    age: 29,
+    // Needs to be string until the API is updated
+    age: '29',
     asl: 'Yes',
+    role: 'manager'
   })
 })
 
+IVP-41: Add More Fields/Update Age About Route
```
Comment (simpsbri) [inline index.js]: Is this intended to be a comment?

File: index.js [hunk 2]
```diff
@@ -110,12 +110,17 @@ app.get('/about', (req, res) => {
   // Sends HTTP response
   // https://expressjs.com/en/5x/api.html#res.send
   res.status(200).json({
+    id: 220395259
```
Comment (aidanho1188) [inline index.js]: There should be a comma after this field.

**Review by simpsbri — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (aidanho1188) [inline index.js]: Leftover?

Comment (dandigangi) [reply index.js]: Nope, needs to be removed. Accident.

Comment (dandigangi) [reply index.js]: Yup

**Review by simpsbri — APPROVED** [review]
lgtm

**Review by aidanho1188 — APPROVED** [review]
Looking good!

**Pull request merged**